### PR TITLE
You can now rename food with a pen.

### PR DESCRIPTION
**Pull request by Iamgoofball**
write "DICKS" on a burger with a sharpie today

### CODE REVIEW

File: code/modules/food&drinks/food/snacks.dm [hunk 1]
```diff
@@ -127,6 +127,11 @@
 
 
 /obj/item/weapon/reagent_containers/food/snacks/attackby(obj/item/weapon/W, mob/user)
+	if(istype(W,/obj/item/weapon/pen))
+		var/n_name = copytext(sanitize(input(usr, "What would you like to name this dish?", "Food Renaming", null)  as text), 1, MAX_NAME_LEN)
```
Comment (tkdrg) [inline code/modules/food&drinks/food/snacks.dm]: You can just do `var/n_name = stripped_input(user,"What would you like to name this dish?","Food Renaming","",MAX_NAME_LEN)`

File: code/modules/food&drinks/food/snacks.dm [hunk 2]
```diff
@@ -127,6 +127,11 @@
...
+		if((loc == user && user.stat == 0))
```
Comment (tkdrg) [inline code/modules/food&drinks/food/snacks.dm]: Can you add a feedback message as an `else` in case this if fails?

**Comment by phil235** [comment]
Meh, I don't like that any assistant can ruin the chef's work.

**Comment by Iamgoofball** [comment]
1. they can always change the name back, fun fact of the day, you start with a pen in your PDA. get it out with an object verb.
2. it's just a name, the food is still good.

**Comment by paprka** [comment]
If it's 'just a name' why do we need this feature when it's always used by griefers or people trying to make bad jokes

**Comment by Iamgoofball** [comment]
because what if the chef wants to name his burger "Quarter-Pounder Supreme" and have it show up as that when eaten? if you do a label it's just "super bite burger (Quarter-Pounder Supreme)"

**Comment by paprka** [comment]
Maybe that's evidence that our recipes suck if chefs don't like the names?

**Comment by TZK13** [comment]
Basically I like the idea of being able to name food, I don't like the idea of anybody being able to name the food the cook puts out. You can do nice gimmicks via naming food copyrighted shit like big macs and whoppers.

**Comment by Cheridan** [comment]
yeah, just give him a hand labeller or some other method

**Pull request closed without merging**